plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
         "SystemCheckMemoryLimit": "Memory limit",
-        "SystemCheckMemoryLimitHelp": "On a high traffic website, the archiving process may require more memory than currently allowed. If necessary, change the memory_limit directive in your php.ini file.",
+        "SystemCheckMemoryLimitHelp": "On a high-traffic website, the archival process may require more memory than currently allowed. If necessary, change the memory_limit directive in your php.ini file.",
         "SystemCheckMemoryNoMemoryLimitSet": "There is no memory limit set",
-        "SystemCheckNoErrorsOrWarnings": "There are no errors or warnings",
-        "SystemCheckViewFullSystemCheck": "View the full system check report",
+        "SystemCheckNoErrorsOrWarnings": "No errors or warnings",
+        "SystemCheckViewFullSystemCheck": "View the full system-check report",
         "SystemCheckOpenURL": "Open URL",
-        "SystemCheckOpenURLHelp": "Newsletter subscriptions, update notifications, and one-click updates requires the \"curl\" extension, allow_url_fopen=On, or fsockopen() enabled.",
+        "SystemCheckOpenURLHelp": "Newsletter subscriptions, update notifications, and one-click updates requires the \"cURL\" extension, allow_url_fopen=On, or fsockopen() enabled.",
         "SystemCheckOtherExtensions": "Other extensions",
         "SystemCheckOtherFunctions": "Other functions",
-        "SystemCheckPageSpeedDisabled": "PageSpeed disabled",
-        "SystemCheckPageSpeedWarning": "We recommend disabling the PageSpeed Module in your web server %s: PageSpeed was reported to cause several issues with Matomo, such as: broken Pages reports, broken Row Evolution, and other issues which are hard to troubleshoot. Please disable mod_pagespeed on this server.",
-        "SystemCheckPackHelp": "The pack() function is required to track visitors in Matomo.",
-        "SystemCheckParseIniFileHelp": "This built-in function has been disabled on your host. Matomo will attempt to emulate this function but may encounter further security restrictions. Tracker performance will also be impacted.",
-        "SystemCheckPdoAndMysqliHelp": "On a GNU\/Linux server you can compile php with the following options: %1$s In your php.ini, add the following lines: %2$s",
+        "SystemCheckPageSpeedDisabled": "PageSpeed is turned off",
+        "SystemCheckPageSpeedWarning": "It is recommended to turn off the PageSpeed module on your %s web server: PageSpeed causes several issues with Matomo, such as: broken pages reports, broken row evolution, etc. Please turn off mod_pagespeed on this server.",
+        "SystemCheckPackHelp": "The 'pack()' function is required to track visitors in Matomo.",
+        "SystemCheckParseIniFileHelp": "This built-in function has been turned off on your host. Matomo will attempt to emulate it, but may encounter further security restrictions. Tracker performance will be impacted.",
```

Comment:
Suggest reverting as "disabled" is more accurate here.

Reply:
I know it is technically more stringent, but if it can't be turned on _and_ disabled in a meaningful sense, this is less complex.